Source/Maestro.H

```diff
@@ -13,18 +13,18 @@
 #include <AMReX_MultiFabUtil.H>
 #include <AMReX_ParmParse.H>
 #include <AMReX_PlotFileUtil.H>
-#include <network.H>
-#include <eos.H>
-#include <conductivity.H>
 #include <BaseState.H>
 #include <BaseStateGeometry.H>
+#include <conductivity.H>
+#include <eos.H>
```

Comment:
I get compilation errors in `conductivity.H` when the files are included in this order. The code compiles properly when `eos.H` is included before `conductivity.H`.

Reply:
This is a problem with Microphysics and means that conductivity.H is missing some include statements that are required. I'll have a look and see if I can fix it.